blog/admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Group
+
+from .models import User, Commentary, Post
+
+
+admin.site.unregister(Group)
+
+
+@admin.register(User)
+class UserAdmin(UserAdmin):
+    list_display = UserAdmin.list_display
+    search_fields = ["username", "first_name", "last_name", ]
+
+
+@admin.register(Commentary)
+class CommentaryAdmin(admin.ModelAdmin):
+    list_display = ["user", "post", "created_time", ]
+    search_fields = ["user", ]
+    list_filter = ["post", "user", ]
+
+
+@admin.register(Post)
+class PostAdmin(admin.ModelAdmin):
+    list_display = ["owner", "title", "created_time", ]
+    search_fields = ["owner", "title", ]
+    list_filter = ["owner", ]
```

Comment:
Totally not obligatory, but mostly people prefer to write code in similar cases by same scenario, so in this case we have something like create, update, delete, so we expect to have all regarding `register` on top, while unregister on bot.